style.css

```diff
@@ -0,0 +1,585 @@
+.about {
+  background-color: #fff;
+  border-radius: 0 8em 0 0;
+  margin: 8em 0 0 0;
+}
+
+.headline-skill-container {
+  padding: 5em 7% 0 7%;
+}
+
+.about-heading-small {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.about-skills-container {
+  display: flex;
+  flex-direction: column;
+  padding: 2em 1em;
+  width: 100%;
+}
+
+.about-skills-container > hr {
+  height: 2px;
+  background: #dfe1e6;
+  border: none;
+}
+
+.about-skill-heading {
+  display: flex;
+  justify-content: space-between;
+  padding: 2em 1em 2em 0;
+}
+
+.about-skill-body {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  background-color: #f7f7f9;
+  border-radius: 0.5em;
+  margin: 0.5em 0;
+  padding: 0.5em;
+}
+
+.about-skill-text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  padding-left: 1em;
+  line-height: 20px;
+  color: #253858;
+}
+
+/* About Section Ends Here */
+
+/* Contact Me starts here */
+.contact-me-form-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  margin: 6em 0 0 0;
+  padding: 3em;
+  border-radius: 5em 0 0 0;
+  background-color: #6070ff;
+  background-image: url(./assets/contact\ form\ shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.contact-me-form-heading {
+  text-align: center;
+  margin-top: 0.75em;
+  font-family: inherit;
+  color: #fff;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 52px;
+}
+
+.contact-me-form-paragraph {
+  text-align: center;
+  margin-top: 0.75em;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+}
+
+.contact-me-form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0.75em;
+}
+
+.contact-me-form > input:focus,
+textarea:focus {
+  outline: none !important;
+  border: 1px solid #f7f7f9;
+  box-shadow: 0 0 2px #f7f7f9;
+}
+
+.contact-me-form-text-input {
+  margin-top: 0.75em;
+  padding: 0.75em;
+  border-radius: 0.75em;
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.contact-me-form-button {
+  align-self: flex-start;
+}
+
+/* Contact me ends here */
+
+@media screen and (min-width: 768px) {
+  /* Header Section Styling Starts Here */
+
+  header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 5em;
+    box-shadow: 0 0.25px 10px 0 rgba(0, 0, 255, 0.2);
+  }
+
+  .hamberger {
+    display: none;
+  }
+
+  .nav-menu-button {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .headline-main {
+    margin-top: 5em;
+    background-image: url(./assets/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    left: 34.6px;
+  }
+
+  .headline-heading {
```

Comment:
- [ ] Kindly add more width to your 'about myself' title to appear on a single line as in the given design:
- yours: 
![image](https://user-images.githubusercontent.com/91486791/235896195-121c260d-7f9b-4193-97be-ff53d95f7ea3.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/235896251-cf31cd8d-158a-47a5-aece-de6d49daa72e.png)

Reply:
Also, I made changes to this also thanks for the review.

Reply:
I removed the <br> tag and it's now fixed.